src/api/cli/integration/integration_test.go

```diff
@@ -288,12 +281,23 @@ func doLiveOrStable(t *testing.T) bool {
 func rpcAddress() string {
 	rpcAddr := os.Getenv("RPC_ADDR")
```

Comment:
why dont we make this `API_ADDRESS` thereby saying that its the same address for the `JSON RPC API` and the `REST API`

Reply:
trying to minimize changes